com.unity.robotics.urdf-importer/Runtime/RosSharpDefinitions/Sensor.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml;
+using System.Xml.Linq;
+using System;
+using System.Numerics;
+
+
+namespace RosSharp.Urdf
+{
+    public class Sensor
+    {
+        public string name;
+        public string type;
+        public string topic;
+        private static string baseKey = "sensor";
+        private static string attributeDelimit = "@";
+        private static string elementDelimit = "/";
+        public int updateRate;
+        public Dictionary<string,string> elements;
```

Comment:
should we follow the naming conventions in https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs ?